core-bundle/src/Util/SimpleTokenParser.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Util;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\LogLevel;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+
+class SimpleTokenParser implements LoggerAwareInterface
+{
+    use LoggerAwareTrait;
+
+    /**
+     * @var ExpressionLanguage
+     */
+    private $expressionLanguage;
+
+    public function __construct(ExpressionLanguage $expressionLanguage)
```

Comment:
Is there no interface to typehint against?

Reply:
No, I don't think so.